### DISCUSSION TITLE
Error: Cannot use 'in' operator to search for 'toDate' in 2024-06-07

### DISCUSSION THREAD

**Discussion by sebastianjammer** [AntlerVC/firetable#1549] [Support / Q&A]
I want to load a date string that I get from a Rest API into a buildship table date column. The date is represented as in the column configuration "yyyy-MM-dd". Unfortunately it comes back with this error.
A similar problem occurs if I use the notation Date(stringVariable).

**Comment by brisring** [comment]
I have the same error!
